Frontend/src/components/Onboarding.tsx

```
@@ -1,37 +1,1487 @@
+            )}
+          </div>
+        ))}
+      </div>
+      {pricingError && <div className="text-red-500 text-sm mt-2">{pricingError}</div>}
+    </div>
+  );
+
+  // Step 5: Profile Picture Upload (new step)
+  const handleProfilePicChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setProfilePicError("");
+    if (e.target.files && e.target.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)